app.js

```diff
@@ -2,9 +2,10 @@ const express = require("express");
 
 const {
   getTopics,
-  getArticleById,
```

Comment:
How come you've taken this out?

Reply:
Ah, I think I see now. I'd like you to separate these concerns, please - so getArticles and getArticleById should be separate functions.